.gitlab-ci.yml

```diff
@@ -36,8 +36,7 @@ default:
   before_script:
     - apt-get update && apt-get install -y git clang-format pcregrep
   tags:
-    # QA-866: Ubuntu 24.04 image cannot properly run on our private runners.
-    - mender-qa-worker-generic-light
+    - hetzner-podman
```

Comment:
Please update the `default` instead (line 27) and remove the `tags` key from all these jobs.